Templates/VSIX/Project Templates/cpp-winui-template/WinUI3TemplateCpp.vcxproj

```diff
@@ -156,6 +147,9 @@
       <DependentUpon>MainWindow.xaml</DependentUpon>
     </ClCompile>
     <ClCompile Include="$(GeneratedFilesDir)module.g.cpp" />
+    <AppxManifest Include="Package.appxmanifest">
+      <SubType>Designer</SubType>
+    </AppxManifest>
```

Comment:
This isn't necessary.

Reply:
When I removed this, the Package.appxmanifest doesn't show up in my project: 
![image](https://user-images.githubusercontent.com/69016571/142051710-6e37f393-4f78-44e6-90e7-2d12c88f2b4a.png)

Reply:
Odd, that doesn't match my experience with the official WinAppSdk project template (note that [it](https://github.com/microsoft/WindowsAppSDK/blob/main/dev/VSIX/ProjectTemplates/Desktop/CppWinRT/SingleProjectPackagedApp/ProjectTemplate.vcxproj) doesn't explicitly include `Package.appxmanifest` since `Microsoft.Build.Msix.Packaging.targets` default-includes it if `WindowsPackageType` is `MSIX`).